Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/README.md

```diff
@@ -0,0 +1,10 @@
+## This demonstrates how to post a request to Azure Functions.
+
+### How to run the project:

```

Comment:
How to run the project:
1.	Download the **Azure Functions sample** and deploy it to Azure.
2.	Download the **console application** project.
3.	Open the solution file in Visual Studio.
4.	Go to the Azure portal.
5.	Locate and click on the App Service option.
6.	Select the appropriate App Service.
7.	Proceed to the Functions tab within the service.
8.	Copy the URL link from the **Get Function URL**.
9.	Run the console application in local.
10.	Paste the URL link into the console application and press the Enter button.
11.	The output PDF document will be generated by Azure Functions.

Reply:
Updated